svc/PrefService.js

```diff
@@ -101,6 +101,16 @@ export class PrefService {
         this.pushPendingBuffered();
     }
 
+    /**
+     * Restore a preference to its default value.
+     * @param key
+     */
+    unset(key) {
+        // TODO: round-trip this to the server as a proper unset?
```

Comment:
Looks like we have a method on PrefService.groovy that's ready to use, although we would need to expose via a controller and coordinate with a Hoist-Core release. To be honest I am not super-motivated to add that to the mix right now.